docs/progress.md

```diff
@@ -36,8 +36,12 @@ Reporting progress works as follows:
 * format of the progress must be `N/M`
 
 The executor will read this file every 3s and if there was an update,
-patch the pod annotations with `workflows.argoproj.io/progress: N/M`.
-The controller picks this up and writes the progress to the appropriate Status properties.
+patch the matching `WorkflowTaskResults` resource with the value `N/M`.
+The controller picks this up and writes the progress to the appropriate Status properties
+every 1m.
```

Comment:
Thanks for this change. [It looks like](https://github.com/argoproj/argo-workflows/blob/main/workflow/executor/executor.go#L862) it's both patching annotations and patching `WorkflowTaskResults` with this progress information, right?

Reply:
Ahh, you're right. I looked at the code too quickly. It's a backup, as you mention in your other comment.